x-pack/legacy/plugins/siem/public/components/timeline/query_bar/index.tsx

```diff
@@ -35,6 +34,7 @@ export interface QueryBarTimelineComponentProps {
   browserFields: BrowserFields;
   dataProviders: DataProvider[];
   filters: Filter[];
+  filterManager: FilterManager;
```

Comment:
instead of passing the props, can we use the timeline context inside of this component

Reply:
It might simplify code/test if we can do like that